en/includes/guides/users/attributes/manage-attributes.md

```diff
@@ -93,3 +93,194 @@ To update the properties of a user attribute:
 6. Go to the **Additional Properties** tab and add additional properties that can be used when writing an extension.
 
     ![Edit additional properties]({{base_path}}/assets/img/guides/organization/attributes/edit-attributes-additional-properties.png){: width="500" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+{{ user_attribute_change_verification }}
+
+{% if product_name == "WSO2 Identity Server" %}
+### Try it out
+
+You can try out the follwoing flows using the provided cURLs:
+
+- Update mobile number
+
+    === "Request"
+        ```
+        curl -v -k --user [username]:[password] -X PATCH
+        -d '{
+            "schemas":[],
+            "Operations":[{"op":[operation], 
+            "value":{[attributeName]:[attribute value]}}]
+        }' 
+        --header "Content-Type:application/json" 
+        https://localhost:9443/scim2/Users/[user ID]
+        ```
+
+    === "Sample Request"
+        ```
+        curl -v -k --user bob:pass123 -X PATCH 
+        -d '{
+            "schemas":["urn:ietf:params:scim:api:messages:2.0:PatchOp"],
+            "Operations":[
+                {
+                    "op":"replace",
+                    "value":{
+                        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User":     {"verifyMobile": "true"},
+                        "phoneNumbers":[{"type":"mobile","value":"0123456789"}]
```

Comment:
Let's format

Reply:
https://github.com/wso2/docs-is/pull/4545/commits/0e6311ec6ed82441ff311dc3555f42d305631f99